src/scripts/core/services/curve.ts

```diff
@@ -46,6 +71,7 @@ export class Curve {
         return Math.PI * Math.pow(this.getRadius(), 2);
     }
 
+    //using known degree, get grid position of a point
     public getPointOnCircumference(degrees: number): [number, number] {
```

Comment:
When commenting functions it can be helpful to use the format `/** comment */`. This will show the comment in intellisense providing that useful information at the point of using the function to help the devs out.

<img width="270" alt="Screenshot 2023-08-25 at 4 50 56 PM" src="https://github.com/KieranWoodTombola/Match-Three/assets/41992522/bfe1a640-e5b9-4f9f-9bd9-c6ce4f38bcae">

<img width="367" alt="Screenshot 2023-08-25 at 4 51 24 PM" src="https://github.com/KieranWoodTombola/Match-Three/assets/41992522/bf87532e-5aa1-4d72-9e10-f4e91edc2faa">

Reply:
It also allows you to spread the comment across multiple lines, as well as insert additional things such as @param which let's you describe what a parameter is for, letting you customise what intellisense shows when invoking said function. Similarly you can give a type to the param by using {} notation. Not always handy as the name of a parameter can suffice, but it can allow for a little extra description where needed.
<img width="489" alt="image" src="https://github.com/KieranWoodTombola/Match-Three/assets/13646039/165bf20a-61c7-414d-b0c7-ad5456831410">
<img width="327" alt="image" src="https://github.com/KieranWoodTombola/Match-Three/assets/13646039/6184d852-08aa-44d0-9d55-2b5d2bd044b2">